Tasks/KubernetesManifestV0/task.json

```diff
@@ -0,0 +1,228 @@
+{
+    "id": "DEE316A2-586F-4DEF-BE79-488A1F503DEE",
+    "name": "KubernetesManifest",
+    "friendlyName": "Manifest based Kubernetes deployments",
+    "description": "Manifest based deployments to kubernetes",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=851275)",
+    "category": "Deploy",
+    "preview": true,
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "demands": [],
+    "groups": [
+        {
+            "name": "config",
+            "displayName": "Config",
+            "isExpanded": true
+        },
+        {
+            "name": "kubernetesCluster",
+            "displayName": "Kubernetes Cluster",
+            "isExpanded": true,
+            "visibleRule": "action != bake"
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectionType",
+            "type": "pickList",
+            "label": "Service connection type",
+            "defaultValue": "Azure Resource Manager",
+            "required": true,
+            "options": {
+                "Azure Resource Manager": "Azure Resource Manager",
```

Comment:
Remove ARM from this list as AJ is adding that support in kubernetes endpoint itself.

Reply:
Removed it.